### PR TITLE
docs/lbi: Note non-support of "rootless"

### PR DESCRIPTION
**Pull request by cgwalters**
Point to the podman discussion about `--userns=auto` instead.

### CODE REVIEW

File: docs/src/experimental-logically-bound-images.md [hunk 1]
```diff
@@ -76,6 +76,31 @@ invocation used by bootc. However, many properties used for container registry i
...
 
 At the current time, logically bound images are [not supported by Anaconda](https://github.com/rhinstaller/anaconda/discussions/5197).
+
+## Comparison with default podman systemd units
+
+In the comparison below, the term "floating" will be used for non-logically bound images. These images are often fetched by e.g. [podman-systemd](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html) and may be upgraded, added or removed independently of the host upgrade lifecycle.
+
+### Lifecycle
+
+- **Floating image:** The images are downloaded by the server the first time it starts (requiring networking typically). Commonly tools such as `podman auto-update` can be used to upgrade them independently of the host.
```
Comment (vrothberg) [inline docs/src/experimental-logically-bound-images.md:95]:
```suggestion
- **Floating image:** The images are downloaded by the server the first time it starts (requiring networking typically). Commonly used tools such as `podman auto-update` can be used to upgrade them independently of the host.
```

**Review by vrothberg — COMMENTED** [review]
I think it's worth stating what will happen on rollback and when trying to remove a logically-bound image.

**Comment by cgwalters** [comment]
OK, pushed a few more tweaks

**Review by vrothberg — APPROVED** [review]
Lovely, LGTM

**Pull request merged**